src/Plugin/LicenseStatus.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Kirby\Plugin;
+
+use Kirby\Cms\LicenseStatus as SystemLicenseStatus;
+use Stringable;
+
+/**
+ * License Status
+ *
+ * @package   Kirby Plugin
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class LicenseStatus implements Stringable
+{
+	public function __construct(
+		protected string $value,
+		protected string $icon,
+		protected string $label,
+		protected string|null $theme = null
+	) {
+	}
+
+	/**
+	 * Returns the status label
+	 */
+	public function __toString(): string
+	{
+		return $this->label();
+	}
+
+	/**
+	 * Returns predefined statuses
+	 */
+	public static function defaults(): array
+	{
+		return [
+			'active' => new static(
+				value: 'active',
+				icon: SystemLicenseStatus::Active->icon(),
+				label: SystemLicenseStatus::Active->label(),
+				theme: SystemLicenseStatus::Active->theme()
+			),
+			'demo' => new static(
+				value: 'demo',
+				icon: SystemLicenseStatus::Demo->icon(),
+				label: SystemLicenseStatus::Demo->label(),
+				theme: SystemLicenseStatus::Demo->theme()
+			),
+			'inactive' => new static(
+				value: 'inactive',
+				icon: SystemLicenseStatus::Inactive->icon(),
+				label: SystemLicenseStatus::Inactive->label(),
+				theme: SystemLicenseStatus::Inactive->theme()
+			),
+			'legacy' => new static(
+				value: 'legacy',
+				icon: SystemLicenseStatus::Legacy->icon(),
+				label: SystemLicenseStatus::Legacy->label(),
+				theme: SystemLicenseStatus::Legacy->theme()
+			),
+			'missing' => new static(
+				value: 'missing',
+				icon: SystemLicenseStatus::Missing->icon(),
+				label: SystemLicenseStatus::Missing->label(),
+				theme: SystemLicenseStatus::Missing->theme()
+			),
+			'unknown' => new static(
+				value: 'unknown',
+				icon: 'question',
+				label: 'Unknown license',
+				theme: 'passive'
+			),
+		];
+	}
```

Comment:
`Suggestion`:  This doesn't look very DRY to me, so I had a though:
- Get rid of `::defaults()`
- Add `unkown` as case to the system `LicenStatus`
- And then in `LicenseStatus::from()` we could to

```php
$status = SystemLicenseStatus::fromString($status ?? 'unknown') ?? SystemLicenseStatus::unknown;
return new static(
    value: $status->value,
    icon: $status->icon(),
    label: $status->label(),
    theme: $status->theme()
)
```

Reply:
Love this! I just implemented it.